nimcfg.json

```diff
@@ -0,0 +1,23 @@
+{
+    "comments": {
+        "lineComment": "#",
+        "blockComment": [ "#[", "]#" ]
+    },
+    "brackets": [
+        ["[", "]"],
+        ["(", ")"]
+    ],
+    "autoClosingPairs": [
```

Comment:
Is it really deprecated if it's inside a json configuration. Because the source codes states: 
```javascript
/**
* **Deprecated** Do not use.
*
* @deprecated * Use the the autoClosingPairs property in the language configuration file instead.
*/
```
That's the reason why I created a json language config file anyway. And [here](https://code.visualstudio.com/docs/extensionAPI/extension-points#_contributeslanguages) in the official manual they still recommend `autoClosingPairs` in json configs. Also e.g. the Python extension(integrated into VSCode) depends on `autoClosingPairs`, otherwise they would have the same problem.